modules/ROOT/pages/api-index.adoc

```diff
@@ -1,5 +1,166 @@
+--
+
+[.card.card-index]
+--
+xref:api-glossary.adoc[[.card-title]#REST API glossary# [.card-body.card-content-overflow]#pass:q[A few things to know to best use the REST API]#]
+--
+
+[.card.card-index]
+--
+xref:rest-api-authentication.adoc[[.card-title]#REST Authentication# [.card-body.card-content-overflow]#pass:q[Login to and logout from Bonita, the prerequisites to call the REST API]#]
+--
+
+[.card.card-index]
+--
+xref:rest-api-extensions.adoc[[.card-title]#REST API extensions overview# [.card-body.card-content-overflow]#pass:q[Introduction and example]#]
+--
+
+[.card.card-index]
+--
+xref:rest-api-extension-archetype.adoc[[.card-title]#Create a REST API extension# [.card-body.card-content-overflow]#pass:q[Tools to develop and test your REST API Extensions]#]
+--
+
+[.card-section]
+== REST API per project element
+
+[.card.card-index]
+--
+xref:application-api.adoc[[.card-title]#Application API# [.card-body.card-content-overflow]#pass:q[Learn how to retrieve the current deployed application definitions]#]
+--
+
+[.card.card-index]
+--
+xref:access-control-api.adoc[[.card-title]#BDM Access Control API# [.card-body.card-content-overflow]#pass:q[Learn how to get the BDM Access Control status (lastUpdatedBy, lastUpdateDate…)]#]
+--
+
+[.card.card-index]
+--
+xref:bdm-api.adoc[[.card-title]#BDM API# [.card-body.card-content-overflow]#pass:q[Learn how to query the Business data]#]
+--
+
+[.card.card-index]
+--
+xref:bpm-api.adoc[[.card-title]#BPM API# [.card-body.card-content-overflow]#pass:q[Learn how to retrieve activities and subprocesses, or to perform maintenance tasks]#]
+--
+
+[.card.card-index]
+--
+xref:customuserinfo-api.adoc[[.card-title]#Custom User Information API# [.card-body.card-content-overflow]#pass:q[Manage your specific user information]#]
+--
+
+[.card.card-index]
+--
+xref:form-api.adoc[[.card-title]#Form API# [.card-body.card-content-overflow]#pass:q[Learn how to map a form to a process or a task]#]
+--
+
+[.card.card-index]
+--
+xref:identity-api.adoc[[.card-title]#Identity API# [.card-body.card-content-overflow]#pass:q[Learn how to manage the organization]#]
+--
+
+[.card.card-index]
+--
+xref:platform-api.adoc[[.card-title]#Platform API# [.card-body.card-content-overflow]#pass:q[Learn how to manage Bonita platform-level resources]#]
+--
+
+[.card.card-index]
+--
+xref:portal-api.adoc[[.card-title]#Portal API# [.card-body.card-content-overflow]#pass:q[Learn how to access and manage pages, profiles, profile entries, profile members, and themes]#]
+--
+
+[.card.card-index]
+--
+xref:system-api.adoc[[.card-title]#System API# [.card-body.card-content-overflow]#pass:q[Learn how to get informations on the current session and on localisation / translation.]#]
+--
+
+[.card.card-index]
+--
+xref:tenant-api.adoc[[.card-title]#Tenant API# [.card-body.card-content-overflow]#pass:q[Learn how to show the tenant status or install / update the BDM]#]
+--
+
+
+[.card-section]
+== Engine API - the basics
```

Comment:
Both REST and JAVA API's are provided by the engine. Calling the Java API's 'engine API' is inaccurate and incomprehensible for developers.

```suggestion
== Java API - the basics
```